config/chime_science_run_gpu_drop_frames.yaml

```diff
@@ -0,0 +1,3521 @@
+##########################################
+#
+# chime_science_run_gpu_drop_frames.yaml
```

Comment:
I'm not sure this should be in the main `config/` directory, but it also shouldn't be in `config/tests` since that's for larger GPU unit tests. 

Since this looks very similar to the setup for the `valve` based drops, I would just remove this config file, but add a comment in the main config files with the drops, that the stages names can be switched `valve` to `TestDataGenMissingFrames` to run this simulation.  Otherwise there is a risk this will just become out of data with the main config if we ever want to run it again.

Reply:
How about I remove it when I add valves to the science configs?

Reply:
Sure.